src/api/liveSpecsExt.ts

```diff
@@ -284,12 +284,14 @@ const getLiveSpecsByConnectorId = async (
         .eq('connector_id', connectorId)
         .eq('spec_type', specType);
 
-    if (prefixFilter) {
-        queryBuilder = queryBuilder.not(
-            'catalog_name',
-            'ilike',
-            `${prefixFilter}/%`
-        );
+    if (prefixFilters && prefixFilters.length > 0) {
+        prefixFilters.forEach((prefix) => {
+            queryBuilder = queryBuilder.not(
+                'catalog_name',
+                'ilike',
+                `${prefix}/%`
+            );
+        });
     }
```

Comment:
This is good with me. 

Thinking... I wonder if there is any impact of handling it like this vs using the `or` function.